doc/source/whatsnew/v0.15.0.rst

```diff
@@ -77,7 +71,7 @@ For full docs, see the :ref:`categorical introduction <categorical>` and the
 .. ipython:: python
     :okwarning:
 
-    df = DataFrame({"id":[1,2,3,4,5,6], "raw_grade":['a', 'b', 'b', 'a', 'a', 'e']})
+    df = DataFrame({"id": [1, 2, 3, 4, 5, 6], "raw_grade": ['a', 'b', 'b', 'a', 'a', 'e']})
```

Comment:
Is this the line too long you mention in the description? You can fix it by simply:
```
df = DataFrame({"id": [1, 2, 3, 4, 5, 6],
                "raw_grade": ['a', 'b', 'b', 'a', 'a', 'e']})
```
Or something similar, if you need to do it somewhere else too.

Reply:
I tried this,but it led to more pep8 errors.

Reply:
If the block is a code-block, you may need to add `...` to the beginning of the continuation file. If you can't get it fixed yourself, leave it the way you think it's correct in the PR, so we can review it, and also see the errors in the CI.